### PR TITLE
Message.replyTo is now an array. Fixes #39

### LINKED ISSUE #39: From and ReplyTo are now multi-valued
Since https://github.com/jmapio/jmap/commit/7b45eb3c2e4983d17a819775238a28746cc7dc01 these two fields are multi-valued.

### PR DESCRIPTION
**Pull request by ddolcimascolo**

### CODE REVIEW

File: test/common/models/Message.js [hunk 1]
```diff
@@ -88,10 +88,10 @@ describe('The Message class', function() {
 
     it('should allow defining optional properties through the opts object', function() {
       var message = new jmap.Message({}, 'id', 'threadId', ['inbox'], {
-        from: {
+        from: [{
```
Comment (thomas-hilaire) [inline test/common/models/Message.js:91]: You could have put one more Emailer there, currently nothing test that we use your new `nthElementOrDefault` for the `from` field.

Comment (ddolcimascolo) [inline test/common/models/Message.js:91]: Actually, `from` being an array does make the code use `nthElementOrDefault`, but I'll add more item, no problem.

File: lib/utils/Utils.js [hunk 2]
```diff
@@ -204,6 +204,24 @@ export default class Utils {
...
+   * or there is not enough elements in the array.
+   *
+   * @param array The `Array` to get the value from.
+   * @param index The index of the desired value.
+   * @param defaultValue The default value to return if the element cannot be found in the array.
+   *
+   * @returns {*} The found value or the given default.
+   */
+  static nthElementOrDefault(array, index, defaultValue) {
+    if (Array.isArray(array) && array.length > index) {
```
Comment (thomas-hilaire) [inline lib/utils/Utils.js]: Giving -1 as the `index` should return `defaultValue`. :trollface:

File: test/common/models/Message.js (hunk 1 above)
Comment (ddolcimascolo) [inline test/common/models/Message.js:91]: Done

File: lib/utils/Utils.js (hunk 2 above)
Comment (ddolcimascolo) [inline lib/utils/Utils.js]: Done

File: test/common/models/Message.js [hunk 3]
```diff
@@ -51,10 +51,10 @@ describe('The Message class', function() {
...
       expect(message.headers).to.deep.equal({});
       expect(message.from).to.deep.equal(jmap.EMailer.unknown());
-      expect(message.to).to.deep.equal([jmap.EMailer.unknown()]);
-      expect(message.cc).to.deep.equal([jmap.EMailer.unknown()]);
-      expect(message.bcc).to.deep.equal([jmap.EMailer.unknown()]);
-      expect(message.replyTo).to.deep.equal(jmap.EMailer.unknown());
+      expect(message.to).to.deep.equal([]);
+      expect(message.cc).to.deep.equal([]);
+      expect(message.bcc).to.deep.equal([]);
+      expect(message.replyTo).to.deep.equal([]);
```
Comment (dbenchi) [inline test/common/models/Message.js:57]: can we have a JSON object called expectedMessage for example with which we can perform deepEqual test?

Comment (ddolcimascolo) [inline test/common/models/Message.js:57]: Not now sorry, I'm lazy

**Pull request merged**